astronomer/providers/amazon/aws/operators/batch.py

```diff
@@ -0,0 +1,104 @@
+"""
+A Deferrable Airflow operator for AWS Batch services
+
+.. see also::
+
+    - http://boto3.readthedocs.io/en/latest/guide/configuration.html
+    - http://boto3.readthedocs.io/en/latest/reference/services/batch.html
+    - https://docs.aws.amazon.com/batch/latest/APIReference/Welcome.html
+"""
+from typing import Any, Dict
+
+from airflow.exceptions import AirflowException
+from airflow.providers.amazon.aws.operators.batch import BatchOperator
+from airflow.utils.context import Context
+
+from astronomer.providers.amazon.aws.triggers.batch import BatchOperatorTrigger
+
+
+class BatchOperatorAsync(BatchOperator):
+    """
+    Execute a job asynchronously on AWS Batch
+
+    .. see also::
+        For more information on how to use this operator, take a look at the guide:
+        :ref:`howto/operator:BatchOperator`
+
+    :param job_name: the name for the job that will run on AWS Batch (templated)
```

Comment:
nit: Please consider removing blank line between param

Reply:
done in [f8c026d](https://github.com/astronomer/astronomer-providers/pull/404/commits/f8c026d9b7dccc417b17fd158282d0b265290ecf)